### ISSUE TITLE
Create a segmentation model diagnostic notebook

### ISSUE THREAD

**Issue by ant0nsc** [microsoft/InnerEye-DeepLearning#293]
* Image sizes
* effect of photometric normalization
* patch sampling thumbnails

Add average Dice to results notebook

**Comment by ant0nsc** [comment]
Can we maybe even add GPU utilization stats, so that people can increase crop size if there is space?

**Comment by ant0nsc** [comment]
We are not always evaluating the model on the full training set, but maybe we can add performance per crop from the last training epochs? Can that give any insights into the quality of the training data?
